quickwit/quickwit-serve/src/ingest_api/rest_handler.rs

```diff
@@ -156,15 +156,12 @@ async fn ingest(
     payload: String,
     mut ingest_service: IngestServiceClient,
 ) -> Result<IngestResponse, IngestServiceError> {
-    let mut doc_batch = DocBatch {
-        index_id,
-        ..Default::default()
-    };
+    let mut doc_batch = DocBatchBuilder::new(index_id);
     for doc_payload in lines(&payload) {
-        add_doc(doc_payload.as_bytes(), &mut doc_batch);
+        doc_batch.ingest_doc(doc_payload.as_bytes());
     }
     let ingest_req = IngestRequest {
-        doc_batches: vec![doc_batch],
+        doc_batches: vec![doc_batch.build()],
```

Comment:
here the behavior doesn't change, but I think `INGEST_METRICS.ingested_num_bytes` will be updated twice, at ingest (here), and when fetching from the queue